theano/tensor/opt.py

```diff
@@ -2920,6 +2988,11 @@ def movable(i):
                 assert tip.type == o_type
                 assert tip.type == mi.owner.inputs[0].type
                 tip = mi.owner.op(tip, *mi.owner.inputs[1:])
+                # Copy over stacktrace from outputs of the original 
+                # "movable" operation to the new operation. 
+                # Julian: Do we want to also include the stacktace of the output (node.outputs[0])?
```

Comment:
Yes, I think the stack trace of `node.outputs[0]` should be included here as well, as it was for ǹew_add`.